libtransact/src/execution/adapter/error.rs

```diff
@@ -16,16 +16,51 @@
  */
 
 use crate::transaction::TransactionPair;
+use std::{error::Error, fmt};
```

Comment:
Typo in commit message, corret -> correct

Reply:
fixed